src/guiBuilder.js

```diff
@@ -0,0 +1,35 @@
+/* globals window */
+
+// TODO : Re-evaluate the api of the gui-builder
+var GUIBuilder = function(buttons) {
```

Comment:
The rule I've always coded with for acronyms is for following letters to be lower case, so for this one `GUIBuilder` should be `GuiBuilder`